src/app/story/storysidebar/storysidebar.component.ts

```diff
@@ -23,4 +23,25 @@ export class StorysidebarComponent implements OnInit {
     }
     return false;
   }
+
+  getResourceLink(): String {
+    switch (this.node.resource_type) {
```

Comment:
Optional: Can you change the comparison to be `this.node.resource_type.toLowerCase()`? That way it could eventually be reusable in other parts of the app that don't have the correct case.